dali/pipeline/operators/reader/loader/loader.h

```diff
@@ -182,7 +181,11 @@ class Loader {
 
   // Give the size of the data accessed through the Loader
   Index Size() {
-    DALI_ENFORCE(loaded_, "Calling Size before data was loaded is an error");
+    if (!loaded_) {
```

Comment:
I think this check is obsolete. If it was called before (during init) it will not be called now. In general, I think that now `loaded_` flag can be removed. It was needed to check, if it is safe to call `Size`, but now it is always safe to do it.

Reply:
Addressed in latest commit